src/components/ButtonPanel.js

```diff
@@ -9,13 +9,13 @@ const ButtonPanel = () => {
     ['1', '2', '3', '+'],
     ['0', '.', '='],
   ];
-
+  const size = sign => (sign === '0' ? 'wide' : '');
   return (
-    <div>
+    <div className="buttonPanel">
       {symbols.map(symbol => (
         <div className="row" key={symbol}>
           {symbol.map(sign => (
-            <Button key={sign} buttonName={sign} />
+            <Button key={sign} color="color" wide={size(sign)} buttonName={sign} />
```

Comment:
> The `color` prop should be used to customize the color of the button. If it is not passed, the button should have an orange background by default.

Pass the color conditionally and have a default color for when no prop is passed for the color prop.

see follow up comments

Reply:
I'll work on this and other comments related to this